spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -774,7 +775,37 @@ Wildcard characters for patterns are determined by the data access provider. For
 
 For relational databases, the logical operator `And` takes precedence over `Or`, meaning that `And` is evaluated on conditions before `Or` when both are specified on the same method. For other database types, the precedence is limited to the capabilities of the database. For example, some graph databases are limited to precedence in traversal order.
 
-=== Special Parameter Handling
+==== Query by Parameters
+
+The Query by Parameters pattern determines the query conditions from the names of the method's parameters that are not of type `Limit`, `Sort`, and `Pageable`. Each query condition is an equality comparison, comparing the parameter value against the value of the entity attribute whose name matches the method parameter name. For embedded attributes, the `_` character is used as the delimiter in the method parameter name. All query conditions are implicitly joined by the `And` operator, such that all must match for an entity to be considered matching.
```

Comment:
I guess we got this discussion somewhere before; annotations such as "Find" could get more flexibility in this case:

https://docs.jboss.org/hibernate/orm/6.3/introduction/html_single/Hibernate_Introduction.html#generated-finder-methods

Reply:
I found it: https://github.com/jakartaee/data/issues/223

Reply:
> I guess we got this discussion somewhere before; annotations such as "Find" could get more flexibility in this case:

It's certainly true that including annotations could add more flexibility here.  However, that's a separate proposal, or separate aspect of a proposal, to consider and discuss separately and is not intended to be addressed by this pull.  The scope of this pull is to specify what is described under your pull #260 where it states "determining the query from the supplied parameters and the `insert`, `update`, `delete`, `find`, `count`, or `exists` prefix. Query by parameters constructs queries based on method parameters and prefixes."

Reply:
Yes, I remember this PR, and that is my point: Does it make sense to have insert and update methods?
Should we update to use "DataAcessObject" instead of Repository?

IMHO: we should remove the insert and update and abstract with "save" for a while.

Reply:
> Yes, I remember this PR, and that is my point: Does it make sense to have insert and update methods? Should we update to use "DataAcessObject" instead of Repository?
> 
> IMHO: we should remove the insert and update and abstract with "save" for a while.

Well, I'm certainly not happy that I wasted all this effort documenting insert/update based on the fact that your pull added them so I thought you were in favor of them.  But if you insist on removing them now, I'll update this pull to do so.

Reply:
@otaviojava I removed insert and update for you: https://github.com/jakartaee/data/pull/266/commits/da6993ebbc64f90d379a79f677b1b6fa71df0db5